x/paloma/keeper/keeper.go

```diff
@@ -103,8 +110,13 @@ func (k Keeper) JailValidatorsWithMissingExternalChainInfos(ctx sdk.Context) err
 
 		sort.Strings(notSupported)
 
+		valAddr, err = utilkeeper.ValAddressFromBech32(k.AddressCodec, val.GetOperator())
```

Comment:
Why are you calling it here again? `valAddr` is already loaded at this point, no?

Reply:
yeah my bad, it might got added while resolving merge conflict. removed and pushed